clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -2116,19 +2118,27 @@ CGCallee ItaniumCXXABI::getVirtualFunctionPointer(CodeGenFunction &CGF,
                                                   llvm::Type *Ty,
                                                   SourceLocation Loc) {
 #ifdef INTEL_SYCL_OPAQUEPOINTER_READY
-  llvm::Type *TyPtr = CGM.GlobalsInt8PtrTy;
+  llvm::Type *PtrTy = CGM.GlobalsInt8PtrTy;
```

Comment:
Can we just name the variable like community code does in both cases?

Reply:
This `PtrTy` is the community code name, `TyPtr` was also the community code before removal.

Reply:
Can we keep the name the same in both #if and #else branches?

Reply:
Yes, we can, but the downside is it will be harder for conflict resolution.
The new name in community is `PtrTy`, so we should use it in `#ifdef`,
if we are change the name in #else from `TyPtr` to `PtrTy`, we need to change all the ref below ( 7 instances).

Then these change will cause conflict when we merging to downstream code , because downstream code is using old names. 

Also the code in `#else` will be removed within weeks or month.

Reply:
But sure, if you insist that we should change the name, I can do it.

Reply:
> Yes, we can, but the downside is it will be harder for conflict resolution. The new name in community is `PtrTy`, so we should use it in `#ifdef`, if we change the name in #else from `TyPtr` to `PtrTy`, we need to change all the ref below ( 7 instances).
> 
> Then these change will cause conflict when we merge to downstream code , because downstream code is using old names.
> 
> Also the code in `#else` will be removed within weeks or month.

Thanks for explaining.   I am okay with the inconsistency then.

Reply:
ok